app/controllers/decks.rb

```diff
@@ -12,11 +13,19 @@
 	deck.cards.each do |card|
 		Guess.create(round: round, card: card)
 	end
-	redirect '/decks/deck.id/cards'
+
+	redirect '/decks/:deck_id/cards'
```

Comment:
You are correct. It should be deck.id

I realized that I had unwittingly been putting routes for filepaths and vice-versa in random places. When I realized it I had Shiraz walk through all of them with me so we could correct them, and then we called it a night. Undoubtedly we missed some.